package.json

```diff
@@ -129,7 +129,7 @@
     "browserify-transform-tools": "^1.7.0",
     "bubleify": "^2.0.0",
     "buffer": "^6.0.3",
-    "canvas": "^2.11.2",
+    "canvas": "^3.1.0",
```

Comment:
Noting that [canvas v3](https://github.com/Automattic/node-canvas/releases/tag/v3.0.0) has dropped support for Node.js 16.x and below and ` "node": ">=18.0.0"` is used in `plotly.js`'s current package system https://github.com/plotly/plotly.js/blob/40f2221f0b2b1714327c5fb9bc0b1109d00aef05/package.json#L24, this change seems good to me.
:100: Thank you :pray: